README.md

```diff
@@ -18,10 +18,11 @@ Launch with:
 
 If you have a [Docker daemon running](https://docs.docker.com/installation/), e.g. reachable on `localhost`, start a container with:
 
-    $ docker run --rm -itP -v "$(pwd):/notebooks" jupyter/notebook
+    $ docker run --rm -it -p 8888:8888 -v "$(pwd):/notebooks" jupyter/notebook
 
 In your browser open the URL `http://localhost:8888/`.
-All notebooks from your session will be saved in the current directory.
+All notebooks from your session will be saved in the current directory. In reality, they are save inside the container at `/notebook`.
```

Comment:
> your example may be well for demonstration purposes. but it's hardly useful for daily usage.

I respectfully disagree. There are quite a few dependencies required by `jupyter`. Getting a nice working copy can sometimes be tricky. Being able to startup a `docker` container instead of debugging the build/install process is a huge asset. Not having to understand all of the ends and outs of `docker` in order to get `jupyter` up and running makes this worthwhile for the average end user.

If your point here is that `docker` is normally used on cluster deployments and the like, then I think that user will know enough about `docker` that they will have no problem changing these options to fit their needs.

> my point is: this is a documentation of the Docker image. that image includes a volume declaration. for non-demonstration-usages it is essential to mention this volume explicitly.

Which it now is.

>> All notebooks from your session will be saved in the current directory. In reality, they are save inside the container at `/notebook`.
> in the case of the example this is not true and a contradiction.

Not true. They are saved in that folder. While the container is up and running one could still find these using `docker exec`. The container is merely blown away on exit instead of merely being stopped.

<br>
I have tried to add a line to address the concern you have raised. If you think I have not done so, please be explicit about what you would like to see instead. I would like this to be resolved in a manner that we can both find satisfactory soon.